x-pack/legacy/plugins/apm/public/components/shared/charts/TransactionCharts/BrowserLineChart.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiTitle } from '@elastic/eui';
+import { TransactionLineChart } from './TransactionLineChart';
+import { TimeSeries } from '../../../../../typings/timeseries';
+import {
+  getMaxY,
+  getResponseTimeTickFormatter,
+  getResponseTimeTooltipFormatter
+} from '.';
+import { getTimeFormatter } from '../../../../utils/formatters';
+
+interface BrowserLineChartProps {
+  series: TimeSeries[];
+}
+
+export function BrowserLineChart({ series }: BrowserLineChartProps) {
+  const maxY = getMaxY(series);
+  const formatter = getTimeFormatter(maxY);
```

Comment:
@smith if you rebase master, you'll have to replace `getTimeFormatter` to `getDurationFormatter`

Reply:
Fixed in rebase.